### ISSUE TITLE
Integration in Phoenix wallet

### ISSUE THREAD

**Issue by robbiehanson** [microchad/bw#8]
Hello Border Wallets team !

We at ACINQ (makers of Phoenix wallet) have been big fans of Border Wallets for some time now. So a few months ago we started researching how Border Wallets might be integrated within the Phoenix app.

#### The missing backup option (= border wallets !)

Phoenix currently offers 2 methods for users to backup their seed:

- Manual backup
- Automatic Cloud backup

The manual backup option, of course, means the user takes 100% responsibility for backing up their recovery phrase. And there are lots of options available to them, including Border Wallets, pen & paper, steel plates, password managers, ...

However there are a lot of users who are scared/intimiated by the manual backup process. This is particularly true for new bitcoin users (who we're trying to onboard). So we also have a 1-tap backup/restore via the user's cloud provider.

The existing options can be thought of as 2 extremes: on one side (_manual_) the user is taking 100% responsibility, and on the other side (*cloud*) they're taking near zero responsibility. 

But we've long felt like there's a missing backup option. What would be ideal would be something in-between the 2 extremes listed above. It would operate as a 2-of-2 scheme, where some data is stored in the cloud, but also requires the user to memorize something. 

And Border Wallets can be used for this purpose ! The data stored in the cloud is the entropy grid. And the user simply needs to memorize their pattern. 

But to achieve our desired result, the process needs to be smooth and easy for the user. We know that Border Wallets can increase the protection for the user. And that advanced bitcoin users can already use the official Border Wallet workflow via the website. But here we want to target the average user and the bitcoin newbie. We want to offer them something that's easy, and entices them to take that next step towards full responsibility. So we started a design & iteration process, where we could run tests with average users.

#### The prototype

It took several design iterations, but we finally landed on something we think works really smoothly for the average user. There are a few key differences from the original Border Wallet flow, for example:

* instead of tapping on grid items to make a pattern, the user can draw with their finger, which they find to be more intuitive
* instead of being forced to make a pattern of exactly 11 or 23 grid items, the user can make a pattern of any size, allowing for more creativity

However, although optimizations were made specifically for mobile apps, the spirit of Border Wallets was maintained. Meaning:

* an entropy grid is generated, in-line with the border wallet specs
* the user combines this with their pattern to generate their 12 or 24 word seed

In other words, we don't consider this to be an alternative version of Border Wallets. But rather an additional workflow. The current workflow is perfect for what is was designed for. This is just a workflow specifically designed for integration within a mobile app.

We detailed our design process [here](https://github.com/ACINQ/phoenix/pull/391), and we were wondering if you would take a look, and let us know what you think.

If you like it, we'd be happy to help formalize the design (_it's pretty minimal_), and could provide working samples for iOS & Android. And you could add it to the Border Wallet website too.
